ktor-client/ktor-client-webrtc/ktor-client-webrtc-rs/common/src/io/ktor/client/webrtc/rs/Utils.kt

```
@@ -0,0 +1,104 @@
+internal suspend inline fun withSdpException(crossinline block: suspend () -> Unit) {
+    try {
+        block()
+    } catch (e: RtcException.SdpException) {
+        throw WebRtc.SdpException(e.v1, e.cause)
+    }
+}
+
+internal suspend inline fun withIceException(crossinline block: suspend () -> Unit) {
+    try {
+        block()
+    } catch (e: RtcException.IceException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)